### DISCUSSION TITLE
Is it possible to create Application with same name in Argocd

### DISCUSSION THREAD

**Discussion by Arjunmeera** [argoproj/argo-cd#14243] [General]
I need to create application with same name in different ns, currently I am getting upsert issue.

**Comment by crenshaw-dev** [comment]
Can you describe the upsert issue?

**Reply by Arjunmeera** [reply]
We have use case for creating Application with same name in different ns. 
Normally It should work With respect to k8's. As per my understanding  the application is getting created in ArgoCD ns. That is the reason we are not able to create application with same name. This is the reason we get upsert issue.  

Do we have any solution for this, could you able to support.
